.gitignore

```diff
@@ -3,3 +3,4 @@
 /monitor/target/
 /monitor/monitor.toml
 /.env
+/docker_jit_monitor/target/
```

Comment:
does this need to be updated?

Reply:
I guess it makes sense to also move the monitor into the docker subdirectory - see the latest commit.